en/docs/develop-components/develop-a-web-application.md

```diff
@@ -26,6 +26,57 @@ You can create a Web Application in Choreo as follows:
 
     Once you create the Web Application component, Choreo will automatically generate a build pipeline for your Single Page Application and deploy it.
 
+    Follow the steps below to manage runtime configurations for your React application at manual deployment.
```

Comment:
Is this applicable only for React?

Reply:
I will adjust it. should be available for all SPA which need to apply a file mount at the time of deployment.